httplib.h

```diff
@@ -6474,6 +6474,11 @@ inline bool Server::handle_file_request(const Request &req, Response &res,
         auto path = entry.base_dir + sub_path;
         if (path.back() == '/') { path += "index.html"; }
 
+        if (detail::is_dir(path)) {
```

Comment:
Then it should also have the opposite part
```suggestion
        if (path.back() == '/') {
          path.pop_back();
          if (detail::is_file(path)) {
             <redirect>
             return true;
          }
          path += "/index.html"; }

        if (detail::is_dir(path)) {
```

Reply:
I don't think it's correct.

<img width="588" alt="image" src="https://github.com/user-attachments/assets/225e8c41-90f6-4501-9302-a60f51c7ac2e">

Reply:
example.com does that.

Reply:
NGINX doesn't redirect `file` to `file/` either, but returns as 404.

<img width="808" alt="image" src="https://github.com/user-attachments/assets/d0aa8061-4c10-4ca7-80db-b77e7e242b62">